src/Symfony/Component/Console/Tests/Command/InvokableCommandTest.php

```diff
@@ -142,6 +144,45 @@ public function testInvalidOptionType()
         $command->getDefinition();
     }
 
+    public function testExecuteHasPriorityOverInvokeMethod()
+    {
+        $command = new class extends Command {
+            public string $called;
+            protected function execute(InputInterface $input, OutputInterface $output): int
+            {
+                $this->called = __FUNCTION__;
+
+                return 0;
+            }
+
+            public function __invoke(): int
+            {
+                $this->called = __FUNCTION__;
+
+                return 0;
+            }
+        };
+
+        $command->run(new ArrayInput([]), new NullOutput());
+        $this->assertSame('execute', $command->called);
```

Comment:
This tests only if it was called, but in theory the other method could be called before

Reply:
Only it was the *last* called. I assumed that was enough to cover the feature.

Reply:
But priority means it should be the first called method, right?

Reply:
In practice, only one of them can be called by Command::run()